### PR TITLE
fix new persistence of CountVectorsFeaturizer

### PR DESCRIPTION
**Pull request by Ghostvv**
**Proposed changes**:
- move tokenize logic to get_message_text
- init `CountVectorizer` with `meta` config in `load`

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: rasa/nlu/featurizers/count_vectors_featurizer.py [hunk 1]
```diff
@@ -291,15 +279,24 @@ def load(
...
         file_name = meta.get("file")
         featurizer_file = os.path.join(model_dir, file_name)
 
         if os.path.exists(featurizer_file):
             vocabulary = utils.json_unpickle(featurizer_file)
 
-            vectorizer = CountVectorizer()
+            vectorizer = CountVectorizer(
+                token_pattern=meta["token_pattern"],
+                strip_accents=meta["strip_accents"],
```
Comment (tmbo) [inline rasa/nlu/featurizers/count_vectors_featurizer.py]: where are these meta properties persisted?

Comment (Ghostvv) [reply rasa/nlu/featurizers/count_vectors_featurizer.py]: `rasa_nlu` persist it separately

Comment (Ghostvv) [reply rasa/nlu/featurizers/count_vectors_featurizer.py]: it is persisted component config

**Comment by Ghostvv** [comment]
I'll add some tests later to prevent this from happening again

**Comment by tmbo** [comment]
https://github.com/RasaHQ/rasa/blob/master/tests/nlu/training/test_train.py#L187 alternatively you can also directly call persist and load on the component and just test that single component

**Comment by Ghostvv** [comment]
@tmbo should be ready for review

File: rasa/nlu/featurizers/count_vectors_featurizer.py [hunk 2]
```diff
@@ -160,36 +159,36 @@ def __init__(
...
 
-    def _tokenizer(self, text):
-        """Override tokenizer in CountVectorizer."""
-
-        text = re.sub(r"\b[0-9]+\b", "__NUMBER__", text)
+    def _get_message_text(self, message):
+        if message.get("spacy_doc"):  # if lemmatize is possible
+            text_tokens = [t.lemma_ for t in message.get("spacy_doc")]
+            # text = " ".join([t.lemma_ for t in message.get("spacy_doc")])
+        elif message.get("tokens"):  # if directly tokens is provided
```
Comment (tmbo) [inline rasa/nlu/featurizers/count_vectors_featurizer.py]: should it rather be the other way around? preferring tokens over spacy?

Comment (Ghostvv) [reply rasa/nlu/featurizers/count_vectors_featurizer.py]: hm, I think we assume that spacy lemma is better than using raw word tokens

Comment (tmbo) [reply rasa/nlu/featurizers/count_vectors_featurizer.py]: :+1:

**Pull request merged**